webui/src/components/ComparePage.js

```diff
@@ -99,7 +99,7 @@ const CompareToolbar = ({repo, refId, compare, refresh}) => {
                     params.set('branch', ref.id);
                     params.delete('commit'); // if we explicitly selected a branch, remove an existing commit if any
                 } else {
-                    params.set('commit', ref.id);
+                    params.set('commit', ref.id.slice(0, 16));
```

Comment:
This one is not the UI, right? it will pass it as a short version of the commit.
if this is the case. I think we should hold just with this one, so we will still work with mvcc.

Reply:
it controls what gets displayed in the button that opens the ref selector after picking a commit. It might possibly control the URI of the resulting request to diff as well